tests/chainer_tests/test_variable.py

```diff
@@ -366,6 +366,67 @@ def test_addgrad_gpu_to_another_gpu(self):
             c = cp.full(3, 30, dtype=np.float32)
         self.check_addgrad(a, b, c)
 
+    def test_debug_print_cpu(self):
```

Comment:
Although it is not a rule but just a convention, we made `check_xxx` method and call it from `test_xxx_cpu` and `test_xxx_gpu` to test some single features with CPU and GPU.
You can see the example e.g. [here](https://github.com/pfnet/chainer/blob/master/tests/chainer_tests/links_tests/connection_tests/test_linear.py#L38).